conf/messages

```diff
@@ -115,13 +113,6 @@ honestyDeclaration.lossOfStaff = of loss of staff essential to the VAT process
 agent.honestyDeclaration.lossOfStaff = my client was affected by loss of staff essential to the VAT process
 honestyDeclaration.other =
 
-# Errors
-# ----------------------------------------------------------
-error.prefix = Error:
```

Comment:
This is being used in fileUploadDialog.scala.html

Reply:
Was a duplicate text. Its still in the document on line 568

Reply:
😮